Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -53,8 +53,11 @@ static void* s_stored_frame = nullptr;
 static u64 s_stored_frame_size = 0;
 static bool s_start_dumping = false;
 
-bool AVIDump::Start(HWND hWnd, int w, int h)
+bool AVIDump::Start(HWND hWnd, int w, int h, DumpFormat format)
 {
+	// Win32 only supports BGR format
+	(void)format;
+
```

Comment:
Fair point, if you leave the Windows signature without the format type, it should be just fine. It has its own signature anyways for the HWND, so you could just drop it.
...unless some later feature (DX12?) could use it. Which might also depend on the planned replacement of VFW with ffmpeg, which might not require the HWND anymore.